cookbooks/cdo-repository/recipes/default.rb

```diff
@@ -3,7 +3,10 @@
 # Recipe:: default
 #
 
-# Sync repo via SSH if key is provided.
+# Chef Managed environments (staging/test/levelbuilder/production) use an SSH key configured in Chef Attributes
+# (cdo-github-access.id_rsa, cdo-github-access.id_rsa.pub) that they use to authenticate to GitHub particularly to commit
+# and push content. Adhocs provisioned with `CHEF_MANAGED=true` (rare) also have these SSH Chef Attributes in addition to a
+# read-only GitHub token that is provisioned on all adhocs from an AWS Secret.
```

Comment:
The information in this comment is repeated three or four times across this PR, and feels particularly out of place here. Can you please pick the one place it feels most relevant to call out this distinction, and only present it there?

Reply:
Most of this documentation belongs at the header of the `cdo-github-access` default, so i've consolidated it there.